src/notebooks/controllers/kernelSource/notebookKernelSourceSelector.ts

```diff
@@ -77,10 +74,6 @@ interface ContributedKernelFinderQuickPickItem extends QuickPickItem {
     kernelFinderInfo: IContributedKernelFinder;
 }
 
-interface LocalJupyterServerQuickPickItem extends QuickPickItem {
```

Comment:
Removed this special case in the picker and moved towards serverUriProvider api.